In [1]:
import numpy as np
import pandas as pd

In [2]:
data = pd.read_csv(r"../input/tvisa-tracking/result.csv")
data.head()

url               file_name  \
0  www.uscis.gov/sites/default/files/err/D12%20-%...  SEP172020_01D12101.pdf   
1  www.uscis.gov/sites/default/files/err/D12%20-%...  SEP172020_02D12101.pdf   
2  www.uscis.gov/sites/default/files/err/D12%20-%...  SEP112020_01D12101.pdf   
3  www.uscis.gov/sites/default/files/err/D12%20-%...  SEP092020_02D12101.pdf   
4  www.uscis.gov/sites/default/files/err/D12%20-%...  SEP092020_01D12101.pdf   

        ID       date                                   decision  \
0  9044770  SEP172020  Appeal of Vermont Service Center Decision   
1  6288405  SEP172020  Appeal of Vermont Service Center Decision   
2  6500174  SEP112020  Appeal of Vermont Service Center Decision   
3  6431583  SEP092020  Appeal of Vermont Service Center Decision   
4  6479121  SEP092020  Appeal of Vermont Service Center Decision   

                                              status  \
0  Form I-914, Application for T Nonimmigrant Status   
1  Form I-914, Application for T Nonimmigrant Status   
2  Form I-914, Application for Family Member of T...   
3  Form I-914, Application for Family Member of T...   
4  Form I-914, Application for T Nonimmigrant Status   

                                        order  is_family  \
0                    The appeal is dismissed.      False   
1                    The appeal is dismissed.      False   
2                     The appeal is rejected.       True   
3                     The appeal is rejected.       True   
4  The decision of the Director is withdrawn.      False   

                                                desc  
0  The Applicant seeks T-1 nonimmigrant classific...  
1  The Applicant seeks T-1 nonimmigrant classific...  
2  The Applicant seeks nonimmigrant classificatio...  
3  the Applicant's Form 1-914, Application for T ...  
4  The Applicant seeks T-1 nonimmigrant classific...

We collected a 188 rows, 9 columns dataset from [non-precedent decisions](https://www.uscis.gov/administrative-appeals/aao-decisions/aao-non-precedent-decisions) (search "Form I-914")


In [3]:
data.shape

(188, 9)

In [4]:
data['order'].value_counts()


The appeal is dismissed.                                                                                                                                  138
The decision of the Director is withdrawn.                                                                                                                 16
The motion to reopen is denied.                                                                                                                            13
The motion to reconsider is dismissed.                                                                                                                      6
The appeal is rejected.                                                                                                                                     3
The motion to reopen is granted.                                                                                                                            2
The matter is remanded for the entry of a new decisi

According to the data, we can see 138 cases are dismissed, 16 cases are withdrawn, and etc...

However, before we can answer questions from clients properly, we need to understand these statements properly then process the data set. We will discuss it during this Friday client meeting. 